catalog/templates/column.html

```diff
@@ -27,112 +27,232 @@
+
+      .. code-block:: postgresql
+
+          WITH data AS (
+            SELECT 1 as id,
+            ST_Buffer(ST_SetSRID(ST_Point({{ col['latlng'][1] }}, {{ col['latlng'][0] }}), 4326), 0.01) the_geom
+          ),
+          meta AS (
+            SELECT OBS_GetMeta(ST_SetSRID(ST_Extent(the_geom), 4326),
+            '[{"numer_id": "{{ col.id }}",
+               "denom_id": "{{ denom_id }}",
+               "normalization": "denominated"}]'
+            , 1, 1) meta
+          FROM data)
+          SELECT id, (data->0->>'value')::{{ col.type }} as rate_estimate
+          FROM OBS_GetData(
+                 (SELECT ARRAY_AGG((the_geom, id)::geomval) FROM data),
+                 (SELECT meta FROM meta))
+
+   {% elif col.aggregate in ('median', 'average') and denom_reltype == 'universe' %}
+   .. tab:: {"tab_name": "Weighted by {{ denom_name }} "}
+
+      Obtain an estimate of "{{ col.name }}" for a polygon
+      using a combination of weighting by area and {{ denom_name }}.
+
+      {% if col.aggregate == 'median' %}Although this measure is a median, the
+      estimate will be a weighted average of medians.{% endif %}
+
+      .. code-block:: postgresql
+
+          WITH data AS (
+            SELECT 1 as id,
+            ST_Buffer(ST_SetSRID(ST_Point({{ col['latlng'][1] }}, {{ col['latlng'][0] }}), 4326), 0.01) the_geom
+          ),
+          meta AS (
+            SELECT OBS_GetMeta(ST_SetSRID(ST_Extent(the_geom), 4326),
+            '[{"numer_id": "{{ col.id }}",
+               "denom_id": "{{ denom_id }}"}]'
+            , 1, 1) meta
+          FROM data)
+          SELECT id, (data->0->>'value')::{{ col.type }} as estimate
+          FROM OBS_GetData(
+                 (SELECT ARRAY_AGG((the_geom, id)::geomval) FROM data),
+                 (SELECT meta FROM meta))
+
+   {% elif denom_reltype == 'denominator' %}
+   .. tab:: {"tab_name": "Per {{ denom_name }}"}
+
+      Obtain value of "{{ col.name }}" per {{ denom_name}} for the
+      smallest available geometry at the selected point.
+
+      .. code-block:: postgresql
+
+          WITH data AS (
+            SELECT 1 as id,
+            ST_SetSRID(ST_Point({{ col['latlng'][1] }}, {{ col['latlng'][0] }}), 4326) the_geom
+          ),
+          meta AS (
+            SELECT OBS_GetMeta(ST_SetSRID(ST_Extent(the_geom), 4326),
+            '[{"numer_id": "{{ col.id }}",
+               "denom_id": "{{ denom_id }}",
+               "normalization": "denominated"}]'
+            , 1, 1) meta
+          FROM data)
+          SELECT id, (data->0->>'value')::{{ col.type }} as val
+          FROM OBS_GetData(
+                 (SELECT ARRAY_AGG((the_geom, id)::geomval) FROM data),
+                 (SELECT meta FROM meta))
+
+   {% endif %}
 
-{{ col.name }} is only available for point lookups.
-{% endif %}
-{% elif col.type|lower == 'geometry' %}
+{% endfor %}
 
-Obtain all "{{ col.name }}" geometries within an area:
+{% if col.aggregate == 'sum' %}
```

Comment:
It is [above](https://github.com/CartoDB/bigmetadata/pull/164/files#diff-d70579600c06a04ff503924c40445bb4R100), isn't it?